miden-lib/src/transaction/inputs.rs

```diff
@@ -185,9 +184,7 @@ fn add_account_to_advice_inputs(
     let code = account.code();
 
     // extend the advice map with the account code data and number of procedures
-    let mut procedures: Vec<Felt> = vec![(code.num_procedures() as u8).into()];
-    procedures.append(&mut code.as_elements());
-    inputs.extend_map([(code.commitment(), procedures)]);
+    inputs.extend_map([(code.commitment(), code.as_elements())]);
```

Comment:
Nit: The comment still mentions the number of procedures - could be updated.

Reply:
Ops, my bad, I overlooked it. Thank you!